custom_model_runner/datarobot_drum/drum/push.py

```diff
@@ -88,13 +83,7 @@ def _push_training(model_config, code_dir, endpoint=None, token=None):
             current_task = "fetching the specified project {}".format(pid)
             project = dr_client.Project(pid)
 
-            # TODO: Update this once the datarobot client is updated
-            payload = dict(user_blueprint_id=user_blueprint_id)
-            current_task = "adding your model to the menu"
-            response = dr_client.client.get_client().post(
-                "projects/{}/blueprints/fromUserBlueprint/".format(pid), data=payload
-            )
-            blueprint_id = response.json()["id"]
+            blueprint_id = dr_client.UserBlueprint.add_to_project(pid, user_blueprint_id)
```

Comment:
are you sure that's correct, this is code [ml-dev observability](https://github.com/datarobot/mldev-observability/blob/main/main/custom_tasks/custom_tasks_sla.py#L208-L216)
```python
    blueprints_added_to_project_menu = UserBlueprint.add_to_project(
        project.id, user_blueprint_id
    )
    blueprint_id = blueprints_added_to_project_menu.added_to_menu[0].blueprint_id
```

Reply:
it returns a `datarobot.models.user_blueprints.models.UserBlueprintAddToProjectMenu`